packages/TorneloScoresheet/src/components/OptionPopup/OptionPopup.tsx

```diff
@@ -0,0 +1,56 @@
+import { View } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import Sheet from '../Sheet/Sheet';
+import { styles } from './style';
+import { colours } from '../../style/colour';
+import PrimaryText, { FontWeight } from '../PrimaryText/PrimaryText';
+import React from 'react';
+import TextIconButton from '../TextIconButton/TextIconButton';
+
+type Option = {
+  icon?: React.FC<SvgProps>;
+  text?: string;
+  onPress: () => void;
+};
+type OptionPopupParams = {
```

Comment:
Our naming convention would have this be `OptionPopupProps`. I'll update our CONTRIBUTING guide to say this